### PR TITLE
added zoomstackRailwayStations query and tests

### PR DESCRIPTION
**Pull request by mysticetii**
Added zoomstackRailwayStations query and test

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -202,6 +202,111 @@ def test_zoomstackNames_build_query_name1(self):
         stripedExpectedFilter = expectedFilter.replace(' ', '').replace('\n', '')
 
         self.assertEqual(stripedPayloadFilter, stripedExpectedFilter)
+    
+    @patch('main.requests.get')
+    def test_zoomstackRailwayStations_one_feature(self, mocked_get):
+        response_data = {"features": ["I'm getting this"]}
+        query = ' { zoomstackRailwayStations(count: 2, propertyName: "Type", literal: "Light Rapid Transit Station") } '
```
Comment (joykuotw) [inline tests/test_main.py:209]: count should be one?

File: tests/test_main.py [hunk 2]
```diff
@@ -202,6 +202,111 @@ def test_zoomstackNames_build_query_name1(self):
...
+        mocked_get.return_value = self.make_mocked_response(response_data)
+        request = self.make_request(query)
+        result = graphqlwfs(request)
+
+        self.assertEqual(result, expected_value)
+    
+    @patch('main.requests.get')
+    def test_zoomstackRailwayStations_empty_filter_parameters(self, mocked_get):
+        response_data = {"features": ["I'm getting this 1", "I'm getting this 2", "I'm getting this also"]}
+        query = ' { zoomstackRailwayStations(count: 2, propertyName: "", literal: " ") } '
```
Comment (joykuotw) [inline tests/test_main.py:232]: count is not correct?

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: tests/test_main.py (hunk 2 above)
Comment (mysticetii) [reply tests/test_main.py:232]: Here count does not matter as the test is to check if filter query is empty or with spaces whether the correct query is built such that empty filter tag is not passed to the server which results in an error response.

File: tests/test_main.py (hunk 1 above)
Comment (mysticetii) [reply tests/test_main.py:209]: Right. I am updating to count: 1

**Review by joykuotw — APPROVED** [review]

**Pull request merged**